### PR TITLE
pycharm: fix creation of virtualenv

### LINKED ISSUE #6884: pycharm@2021.2.2: Failed to create virtual environment if run from current folder
I installed python from scoop, and pycharm.
Then run pycharm from "C:\Users\MYNAME\scoop\apps\pycharm\current\bin\pycharm64.exe", and create a new project and interpreter, then it show Failed to create virtual environment.
If I clean reinstall pycharm and run it from "C:\Users\MYNAME\scoop\apps\pycharm\2021.2.2\bin\pycharm64.exe", it works fine.

\bin in PATH doesn't matter. I have tested remove it from PATH, nothing help.

### PR DESCRIPTION
**Pull request by alealexpro100**
Closes #6884

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by issaclin32 — APPROVED** [review]

**Comment by alealexpro100** [comment]
The issue looks like [this](https://github.com/pypa/virtualenv/issues/194).
Pointing to the real location without any symlinks allows pycharm's virtualenv helper to work correctly.
It can be easily reproduced by running something like this: 
`C:\Users\User\scoop\apps\python\current\python.exe -S C:\Users\User\scoop\apps\pycharm\current\IDE\plugins\python-ce\helpers\virtualenv.pyz C:\Users\User\PycharmProjects\pythonProject1\venv`
In short, running bat file causes direct launch of executable bypassing this bug.

**Comment by rashil2000** [comment]
Oh, thanks for the detailed explanation!
